Gopkg.toml

```diff
@@ -24,34 +24,27 @@
 #   go-tests = true
 #   unused-packages = true
 
-
-[[constraint]]
-  name = "github.com/go-kit/kit"
-  version = "0.7.0"
-
 [[constraint]]
   name = "github.com/opentracing/opentracing-go"
-  version = "1.0.2"
+  version = "1.1.0"
 
 [[constraint]]
   name = "github.com/openzipkin/zipkin-go-opentracing"
   version = "=0.3.2"
 
 [[constraint]]
   name = "github.com/pkg/errors"
-  version = "0.8.0"
+  version = "0.8.1"
 
 [[constraint]]
   name = "github.com/stretchr/testify"
   version = "1.2.2"
 
 [[constraint]]
   name = "github.com/uber/jaeger-client-go"
-  version = "2.14.0"
+  version = "2.19.0"
+
 
-[[constraint]]
-  name = "github.com/apache/thrift"
-  version = "=0.11.0"
```

Comment:
What is the reason behind the removal of this constraint?
It was added to fix the eternal dependency problem : https://github.com/ricardo-ch/go-tracing/pulls?q=is%3Apr+is%3Aclosed